recipes/rapidyaml/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "rapidyaml" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/biojppm/rapidyaml.git
+  git_rev: v{{ version }}
+  # url: https://github.com/biojppm/rapidyaml/archive/refs/tags/v{{ version }}.tar.gz
+  # sha256: 8cad74d721ebab6f6e4909a9622ca27491c3b742782f55db4991a0b9de7422f3
```

Comment:
why you are not using the release file?

Reply:
Dependencies should be built separately and linked dynamically instead of revended and linked statically.

If the project is incompatible with a stable release of the dependency project (the one revended in a submodule), then you can list multiple URL sources and unpack them into the correct folder. You will have to manually update the submodules with each new release.

```
source:
  - url: https://github.com/foo/foo/releases/download/{{ version }}/foo-{{ version }}.zip
    sha256: 3485471defaec4f458a48db45523c87eff13c2a7eb6dec140f606faf13f8fe40
# List the submodules as separate sources only if foo isn't compatible with stable release of bar
  - url: https://github.com/bar/bar/archive/7cc409eeef9de1e7ac3ec01c8b76f640deec36bd.tar.gz
    sha256: 6dc30c9db1fb55e1009f76cafde7a5e0b9de0f14933336437748dfa25d8b3978
    folder: third-party/bar
  - url: https://github.com/spec/spec/archive/7cc409eeef9de1e7ac3ec01c8b76f640deec36bd.tar.gz
    sha256: 6dc30c9db1fb55e1009f76cafde7a5e0b9de0f14933336437748dfa25d8b3978
    folder: third-party/spec
```